doc/nrf/ug_thingy91_gsg.rst

```diff
@@ -85,7 +85,13 @@ The data is transmitted to nRF Cloud.
 
 Before you start using the Thingy:91, it is recommended that you update the application firmware to :ref:`asset_tracker_v2`.
 You must also update the modem firmware.
-You can do this through :term:`Universal Serial Bus (USB)` (MCUboot) or an external debug probe by using the Programmer application.
+You can update the Thingy:91 application and modem firmware through the :term:`Universal Serial Bus (USB)` cable using MCUboot.
+The MCUboot is a secure bootloader used to update applications in case you do not have an external debugger.
+Alternatively, you can use an external debug probe.
```

Comment:
It might make sense to move the note about the debug probe  that is further down (https://github.com/nrfconnect/sdk-nrf/pull/8720/files#diff-90d0e4bb792f25c3e98c1eeb33e528b8d7ff1ea2413b4975771ffeb64ec4fcd0R102) up here.